scio-test/src/main/scala/com/spotify/scio/testing/PipelineTestUtils.scala

```diff
@@ -54,14 +54,6 @@ trait PipelineTestUtils {
     sc.run()
   }
 
-  def tapWithRealContext[T](
-    options: PipelineOptions = PipelineOptionsFactory.create()
```

Comment:
as `runWithLocalOutput` does not take custom options, removed also from the `runWithOutput` API